### PR TITLE
[#73215579228013] Extension timestamp links to keep page

### PR DESCRIPTION
**Pull request by cvializ**
@andrewconner 

screen shot 2015-12-16 at 11 58 55 am

Look at that snazzy underline on hover in the timestamp.

### CODE REVIEW

File: packrat/scripts/thread.js [hunk 1]
```diff
@@ -234,9 +240,13 @@ k.panes.thread = k.panes.thread || function () {
...
   }
 
-  function renderMessage(m) {
+  function renderMessage(keep, m) {
     m.formatMessage = formatMessage.full;
     m.formatAuxData = formatAuxData;
+    m.formatKeepUrl = function () {
+      return 'https://www.kifi.com/k/' + encodeURIComponent(this.title.slice(0, 30)) + '/'+ this.id;
+    };
+    m.keep = keep;
```
Comment (camhashemi) [inline packrat/scripts/thread.js:249]: I would prefer to pass you `keep.path` than to construct this here. do you know what endpoint you get `keep` from?

Comment (cvializ) [inline packrat/scripts/thread.js:249]: hey, thanks : D it's one of the websocket ones, `thread`

Comment (camhashemi) [inline packrat/scripts/thread.js:249]: `get_thread` I'm guessing

Comment (cvializ) [inline packrat/scripts/thread.js:249]: hmm yes that's what the extension sends to get a thread, I think

**Pull request merged**